### DISCUSSION TITLE
🔗 Option to select a different background color for active sticky elements

### DISCUSSION THREAD

**Discussion by Jak1117** [elementor/elementor#16490] [Feature Request]
I would like to be able to have my sticky header menu text and background colors change on scrolling as seen in the video and link below. 

https://www.youtube.com/watch?v=lW8MVPn7y54&ab_channel=LivingWithPixels
https://elementor.com/

**Comment by Kishorchandth** [comment]
@Jak1117 here are the in-depth tutorials I wrote about [sticky header](https://foxscribbler.com/sticky-header-with-elementor/) but yeah having a feature might be helpful for users who don't know how to code.
